pkg/pillar/scripts/device-steps.sh

```diff
@@ -185,6 +185,9 @@ echo "$(date -Ins -u) device-steps: Starting upgradeconverter"
 $BINDIR/upgradeconverter
 echo "$(date -Ins -u) device-steps: upgradeconverter Completed"
 
+echo "$(date -Ins -u) Starting dmesg" >>$PERSISTDIR/log/dmesg.log
+dmesg -T -w --time-format iso >>$PERSISTDIR/log/dmesg.log &
```

Comment:
Can this grow very large?

Reply:
Sorry that code shouldn't be there. Will fix.